modules/pam_env/pam_env.c

```diff
@@ -806,11 +993,8 @@ handle_env (pam_handle_t *pamh, int argc, const char **argv)
 
   retval = _parse_config_file(pamh, ctrl, conf_file);
 
-  if(readenv && retval == PAM_SUCCESS) {
+  if(readenv && retval == PAM_SUCCESS)
     retval = _parse_env_file(pamh, ctrl, env_file);
-    if (retval == PAM_IGNORE)
-      retval = PAM_SUCCESS;
```

Comment:
Why do you remove these two lines?  Is this a bug fix, or is it a change in behaviour?